docs/PullRequests.md

```diff
@@ -0,0 +1,29 @@
+# Pull Requests
+
+## Before opening a pull request
+
+Before opening a pull request, please be sure that:
+
+- your code follows our [contributing guidelines](./ContributionGuidelines.md) and that your [code quality checks](./CodeQuality.md) pass locally.
+- you tested your code on an ios and android and made sure that both are consistent with our visual guidelines.
+- you documented your code. If it is a component, make sure you documented the props and added examples in you sample app and web documentation.
```

Comment:
Should we add something about storybook? Like:

- make sure your documentation is rendering and legible on storybook

Reply:
boa! done in c5e361b4c7b94fd71b8f4c1d35d2f3b453a4bbc3